tests/framework/Optimizers/GeneticAlgorithms/discreteIntWithReplacement/MKmodel.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+
+def run(self,Input):
+  """
+    Evaluate a simple function.
+    @ In, self, object, container
```

Comment:
update docstring, and camelBack for 'Input', I think you need to use a different word for this param, since 'input' is protected.